src/operators/rasterscaling.md

```diff
@@ -0,0 +1,84 @@
+# RasterScaling
+
+The raster scaling operator scales/unscales the values of a raster by a given slope factor and offset.
+This allows to shrink and expand the value range of the pixel values needed to store a raster. It also allows to shift values to all-positive values and back.
+We use the [GDAL](https://gdal.org/index.html) terms of [scale](https://gdal.org/programs/gdal_translate.html#cmdoption-gdal_translate-scale) and [unscale](https://gdal.org/programs/gdal_translate.html#cmdoption-gdal_translate-unscale).
+Raster data is often scaled to reduce memory/storage consumption.
+To get the "real" raster values the unscale operation is applied.
+Keep in mind that scaling might reduce the precision of the pixel values.
+(To actually reduce the size of the raster, use the [raster type conversion operator](./rastertypeconversion.md) and transform to a smaller datatype after scaling.)
+
+The operator applies the following formulas to every pixel.
+
+For _unscaling_ the formula is: `p_new = p_old * slope + offset`.
+
+For _scaling_ the formula is: `p_new = (p_old - offset) / slope`
+
+`p_old` and `p_new` refer to the old and new pixel value. The slope and offset values are either properties attached to the input raster or a fixed value.
+
+An example for Meteosat Second Generation properties is:
+
+- offset: `msg.calibration_offset`
+- slope: `msg.calibration_slope`
+
+## Parameters
```

Comment:
The Example json values should not be surrounded by quotes

Reply:
[965aa0d](https://github.com/geo-engine/docs/pull/33/commits/965aa0d901a75ab6c7485ec452ae76a0e02276c5)

Reply:
the "scale" example value should still have the quotes